### DISCUSSION TITLE
how to install rct2 expansions into openrct2?

### DISCUSSION THREAD

**Discussion by ghost** [OpenRCT2/OpenRCT2#23100] [Q&A] [answered]
Hi

I have bought rct2 cd and 2 expansion cds on a market with old games.
I have installed rct2 cd on my windows pc and openrct2 works now.

How to install the  2 expansion cd inside openrct2?

![RollerCoaster_Tycoon_2_Wacky_Worlds_0000](https://github.com/user-attachments/assets/9e23c28f-238a-4ebf-82dc-b78530e0041c)

![1) Rollercoaster Tycoon 2 Time Twister Box Front](https://github.com/user-attachments/assets/da501871-f3d5-42f0-809f-b5ec85a67a30)

**Comment by telk5093** [comment, marked as answer]
Just install RCT2 and its expansion packs with your CDs, and install OpenRCT2 at the last.
OpenRCT2 is fully compatible with RCT2 and its extension packs.
